plugin_templates/dhcpcontroller/dhcpcontroller.yaml

```diff
@@ -690,3 +690,463 @@ spec:
   selector:
     control-plane: controller-manager
 ---
+apiVersion: v1
+kind: Namespace
```

Comment:
We should move namespace creation/deletion outside of the addon yaml file.

Reply:
node-feature-discovery, dhcp-controller both have namespace definitions in their respective yaml

Reply:
we should fix that for dhcpcontroller as well.. I don't think any other addon is doing that.. During uninstall, namespace cleanup cannot happen before other resources are gone

Reply:
namespace should be templatized and user can deploy kubemacpool in namespace of their choice

Reply:
done